docker-java-shaded/pom.xml

```diff
@@ -0,0 +1,97 @@
+<!--Copy from https://github.com/jenkinsci/docker-plugin/blob/master/docker-java-shaded/pom.xml-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <!-- nb: No parent, otherwise we inherit wrong versions from
+  transient jenkins dependencies. -->
+
+  <groupId>org.jenkins-ci.plugins</groupId>
+  <artifactId>docker-java-shaded-traceability</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-beta-4-SNAPSHOT</version>
+  <name>docker-java shaded for CloudBees Docker Traceability</name>
+
+  <properties>
+    <docker-java.version>1.3.0</docker-java.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>${docker-java.version}</version>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <finalName>${project.artifactId}</finalName>
+    <plugins>
+      <plugin>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>2.3</version>
+        <executions>
+          <execution>
+            <id>shade</id>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <createSourcesJar>true</createSourcesJar>
+          <artifactSet>
+            <includes>
+              <include>org.glassfish*:*</include>
+              <include>org.apache.httpcomponents*:*</include>
+              <include>com.github.docker-java*:*</include>
+              <include>com.google.guava:*</include>
+            </includes>
+          </artifactSet>
+          <relocations>
+            <relocation>
+              <pattern>org.apache.http</pattern>
+              <shadedPattern>shaded.org.apache.http</shadedPattern>
+            </relocation>
+            <relocation>
+              <pattern>com.google.common</pattern>
+              <shadedPattern>shaded.com.google.common</shadedPattern>
+            </relocation>
+            <relocation>
+              <pattern>com.github.dockerjava</pattern>
```

Comment:
I had no time for cleaning and this was copy-pasted to this plugin `<pattern>com.github.dockerjava</pattern>` vs `<include>com.github.docker-java*:*</include>`